secator/tasks/nmap.py

```
@@ -6,11 +6,9 @@
 
 from secator.config import CONFIG
 from secator.decorators import task
-from secator.definitions import (CONFIDENCE, CIDR_RANGE, CVSS_SCORE, DELAY,
-								 DESCRIPTION, EXTRA_DATA, HOST, ID, IP, PROTOCOL, MATCHED_AT, NAME,
-								 OPT_NOT_SUPPORTED, OUTPUT_PATH, PORT, PORTS, PROVIDER,
-								 PROXY, RATE_LIMIT, REFERENCE, REFERENCES, RETRIES, SCRIPT, SERVICE_NAME,
-								 SEVERITY, STATE, TAGS, THREADS, TIMEOUT, TOP_PORTS)
+from secator.definitions import (CIDR_RANGE, DELAY, HOST, IP, OPT_NOT_SUPPORTED,
+								 OUTPUT_PATH, PORTS, PROXY, RATE_LIMIT, RETRIES, SCRIPT,
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)